src/helpers/element-close-call.js

```diff
@@ -4,7 +4,7 @@ import toReference from "./ast/to-reference";
 import iDOMMethod from "./idom-method";
 
 // Returns the closing element's function call.
-export default function elementCloseCall(t, path, plugin) {
+export default function elementCloseCall(t, path, plugin, { components, isComponent }) {
```

Comment:
Almost, I was aiming for a `isComponent` helper function like `iDOMMethod` and similar:

```js
# helpers/is-component.js
export default function isComponent(tag) {
  return /^[A-Z]/.test(tag);
}

# helpers/element-close-call.js
import isComponent from './is-component';

//...
```